code/modules/research/designs/autolathe_designs.dm

```diff
@@ -559,6 +559,14 @@
 	build_path = /obj/item/assembly/voice/noise
 	category = list("initial", "Miscellaneous")
 
+/datum/design/goggles
+	name = "Goggles"
+	id = "goggles"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 100, MAT_GLASS = 100)
+	build_path = /obj/item/clothing/glasses/goggles
+	category = list("initial", "Miscellaneous")
```

Comment:
Are you sure about adding it to the autolathe? It feels a bit out of place and it being on the vendor/loadout should be enough.

Reply:
That is more than prescription glasses, which has more utility and that can be combined with HUDs.
And no need to request another review, i approved the changes as you can see above.

Reply:
There's usually a box or two in medical filled with em, there's enough on the station usually